boilerplate/openshift/golang-osd-operator/csv-generate/common-generate-operator-bundle.py

```diff
@@ -239,6 +251,43 @@ def trim_index(index, kind, item):
     # Get rid of these so we can iterate over what's left at the end
     trim_index(by_kind, kind, 'ALL')
 
+
+csv['spec']['install']['spec']['permissions'] = []
+
+if 'namespace' not in csv['metadata']:
+    raise UndefinedCSVNamespace(OPERATOR_NAME)
+
+# Find namespace of operator from CSV template
+namespace = csv['metadata']['namespace']
+
+if 'Role' in by_kind:
+    filtered_roles = []
+    filtered_rolebindings = []
+    for role in by_kind['Role']:
+        # We assume there is always a rolebinding of defined role
+        if role['metadata']['name'] not in rb_by_role:
+            raise NoAssociatedRoleBinding(role['metadata']['name'], role['metadata']['namespace'])
+        role_binding = rb_by_role[role['metadata']['name']]
+
+        # If the RoleBinding subject is a ServiceAccount in the same namespace as the operator
+        # we add it to CSV. Else it is to be written as-is to the bundle
+        if len(role_binding['subjects']) == 1 and \
+            role_binding['subjects'][0]['kind'] == 'ServiceAccount' and \
+            role_binding['subjects'][0]['namespace'] == namespace:
+                csv['spec']['install']['spec']['permissions'].append(
+                    {
+                        'rules': role['rules'],
+                        'serviceAccountName': role_binding['subjects'][0]['name'] 
+                    }
+                )
+        else:
+            filtered_roles.append(role)
+            filtered_rolebindings.append(role_binding)
+
+    # Overwrite the existing Roles and RoleBindings
+    by_kind['Role'] = filtered_roles
+    by_kind['RoleBinding'] = filtered_rolebindings
```

Comment:
This is what [trim_index](https://github.com/openshift/boilerplate/pull/141/files#diff-e12cb709d3181a1ca667b183451c41bbb410dd055344a17d1c0bc981430fde16R167) is for:
```suggestion
        # Get rid of these so they aren't also copied into the bundle
        trim_index(by_kind, 'Role', role)
        trim_index(by_kind, 'RoleBinding', role_binding
```